kola/tests/misc/nfs.go

```diff
@@ -61,7 +55,23 @@ func NFS(c platform.TestCluster) error {
 		Hostname: "nfs1",
 	}
 
-	m1, err := c.NewMachine(c1.String())
+	mounttmpl = `[Unit]
+Description=NFS Client
+After=network-online.target
+Requires=network-online.target
+After=rpc-statd.service
+Requires=rpc-statd.service
+
+[Mount]
+What=%s:/tmp
+Where=/mnt
+Type=%s
+Options=defaults,noexec
```

Comment:
You need to use the `nfs` type and toggle `nfsvers=4` vs `nfsvers=3` in options. The client and server are free to negotiate version 3 or 4 if with the `nfs` type.